src/Microsoft.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -45,7 +45,7 @@ public class AsymmetricSignatureProvider : SignatureProvider
         };
 
         /// <summary>
-        /// Mapping from algorithm to minimum <see cref="AsymmetricSecurityKey"/>.KeySize when verifying signatures.
+        /// Mapping from algorithm to minimum <see cref="SecurityKey.KeySize"/> when verifying signatures.
```

Comment:
We specifically used AsymmetricSecurityKey here as this is the AsymmetricSignatureProvider.

Reply:
We cannot do something like `<see cref="AsymmetricSecurityKey.KeySize"/>` because the property is defined in the base class and the XML doc doesn't seem to support inheritance. Rolling back this part.